update index.test.js

saszaCodes opened the pull request:
screenshot of tests results:

![image](https://user-images.githubusercontent.com/54599709/124269293-ff22f800-db3a-11eb-8e0d-127e999ed78e.png)

saszaCodes committed:
Create index.test.js

added tests for other functions in index.js

Avuidrauxs reviewed (changes requested):
Add a screenshot of your passing tests to the PR description.

saszaCodes committed:
Added updated filex index.js and index.test.js

![image](https://user-images.githubusercontent.com/54599709/124269293-ff22f800-db3a-11eb-8e0d-127e999ed78e.png)

saszaCodes committed:
Delete index.test.js

saszaCodes commented:
Added the screenshot above and added both updated files index.js and index.test.js instead of adding each in a separate pull request

Avuidrauxs reviewed (approved):
Awesome! Good job
